theme/src/main/java/com/shekharhandigol/LoginScreen.kt

```diff
@@ -19,7 +19,7 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.text.input.PasswordVisualTransformation
 import androidx.compose.ui.unit.dp
 import androidx.navigation.NavController
-import com.shekharhandigol.theme.BothPreview
+import com.shekharhandigol.theme.BothPreviews
 
 @Composable
```

Comment:
replace this with a dummy UI till u work on it. Its confusing to have two login screens.